kvmagent/kvmagent/plugins/virtual_pci_device/gpu/nvidia.py

```
@@ -0,0 +1,545 @@
+            )
+
+            return worker_info
+
+        except (psutil.NoSuchProcess, psutil.AccessDenied, psutil.ZombieProcess, Exception):
+            return None
+
+    def _get_pci_address_by_cuda_index(self, cuda_index):
+        """Get PCI address by CUDA device index"""
+        try:
+            cmd = "nvidia-smi --query-gpu=index,pci.bus_id --format=csv,noheader,nounits"
+            ret, output, err = bash_roe(cmd)
```

Ruff (0.14.5)

Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)